docs/python/environments.md

```diff
@@ -22,6 +22,15 @@ The selected environment is also automatically activated when using the **Python
 
 ## Global and virtual environments
 
+
+> As of version _2018.8.1_, the Python Extension automatically updates environments, this can be disabled, by setting the configuration in `settings.json`:
+
+```python
+{
+    "python.terminal.activateEnvironment": false
+}
+```
+
```

Comment:
Hi--Greg has already asked you to scope your contributions specifically to individual areas, so I believe you'll be resubmitting this PR for the Python change here. When you do, I suggest a simple wording change: "The Python Extension version _2018.8.1_ and later automatically updates environments. To disable automatic updating, add `"python.terminal.activateEnvironment": false` to your `settings.json` file."

Then you don't need the explicit code snippet, which I typically don't include when mentioning other settings like this.